wasmbinding/bindings/query.go

```diff
@@ -53,6 +59,18 @@ type QueryRegisteredQueryResultRequest struct {
 	QueryID uint64 `json:"query_id,omitempty"`
 }
 
+type OracleQuery struct {
+	GetAllCurrencyPairs *oracletypes.GetAllCurrencyPairsRequest `json:"get_all_currency_pairs,omitempty"`
+	GetPrice            *oracletypes.GetPriceRequest            `json:"get_price,omitempty"`
+	GetPrices           *oracletypes.GetPricesRequest           `json:"get_prices,omitempty"`
+}
+
+type MarketmapQuery struct {
+	GetMarketMap   *marketmaptypes.GetMarketMapRequest   `json:"get_market_map,omitempty"`
```

Comment:
I think it's better to rename this to `MarketMap` according to new changes:
https://github.com/skip-mev/slinky/blob/043ce53b6a9ddbab5b7ddfde8fbffebd0b9256ca/proto/slinky/marketmap/v1/query.proto#L16